adr/0001-community-discussions.adoc

```diff
@@ -0,0 +1,70 @@
+- people reported that they don't feel okey posting on quarkus-dev or zulips as it is seems focused on dev work and not so much about community events, jobs, conferences, etc.
+- its hard to monitor as contributor who wants to help answer/ask questions.
+
+How can we improve this situaton and enable the broader community to more easily ask questions and find answers - without it all be relying on just a few Quarkus core contributors?
+
+== Scenarios (optional)
+
+User wants to locate a answer to a question - zulip chats does not show up in google search; stackoverflow might but might not have an answer.
+
+Contributor want to arrange or attend an event around Quarkus - where do he post/look for info on that ?
+
+Contributor/committer are trying to follow new questions and issues - what does he need to track to do that ? today it is alot and it is all different notification sytems and mainly its a big "firehose" with no good ability to filter out what you don't need.
+
+User looking for a job involving Quarkus - where do he go look ?
+
+
+////
+ as well as a lot of social platform presence https://twitter.com/quarkusio/[twitter], https://www.facebook.com/quarkusio/[facebook], https://www.linkedin.com/groups/13789086/[linkedin], https://www.youtube.com/quarkusio[youtube] and https://www.linkedin.com/groups/13789086/[reddit].
+////
+
+== Considered options
+
+* stackoverflow will not go away - thus for this current proposal it does not include Q/A section.
+* "I can't track more places" is a very valid worry; the advantage of discussions is that it shows up in your normal github notifcations which you can mute, filter,etc. like anything else on Github. Thus it gives you more control rather than just adding more noise.
+
+=== quarkusio-community mailing list
+Not proposed since it is just another waterhose of info with no good ability to ignore/consume as needed/wanted.
+
+=== enable discussions on github.com/quarkusio/quarkus-insights
+Considered as quarkus insights is an existing "community" and let us have separate permissons for moderators/triage but some suggested it might be hard to find.
+
+=== enable discussions on github.com/quarkusio/community
+Downside is that labels etc. would need to be kept in sync and that searches in github.com/quarkusio would not show up community/discussions.
+It could show up if searched on github.com/quarkus level though.
+Could be considered as allow us to have separate permissions for moderator/triagers separate from commit access - need to check if https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization[github docs] on permissions levels work for us.
+
+== Decision
+
+Enable GitHub discussions feature on `quarkusio/quarkusio` with the following initial categories:
+
+- Announcements (post only by admins)
```

Comment:
I won't post release announcements in several channels so either they will go there or to quarkus-dev if you open both (+ Twitter). That being said, I don't think it's worth posting release announcements here.

So just be aware: if you open this, don't ask me in two months to also post the release announcements there :).

Reply:
I think we automated that in Mutiny: https://github.com/smallrye/smallrye-mutiny/discussions/categories/announcements. But I don't know if @jponge does it or if it's automatic.

Reply:
github releases can automatically be posted there and be linked to have conversations on the release.

Reminds me of another usecase  i've seen applied - use github discussion as comment source for the blog: https://artsy.github.io/blog/2017/07/15/Comments-are-on/

Not something for day one but having all this in one "bucket" gives some nice simplifications.